Apps/WebClient/src/ClientApp/src/views/faq.vue

```diff
@@ -270,12 +246,12 @@
                                 most immunizations administered by public health
                                 and pharmacies in BC. However, immunizations
                                 administered by family practice or travel
-                                clinics are not recorded in the Registry. Note
-                                that immunizations administered by a pharmacy
-                                may be displayed in your profile as a dispensed
-                                medication rather than an immunization , but
-                                these will be added to the Registry in the
-                                future.
+                                clinics are not recorded in the Registry. 
+                                Immunizations administered by a pharmacy
+                                may be displayed as a dispensed
+                                medication rather than an immunization. 
+                                If you can’t find an immunization record, 
```

Comment:
Do we have a policy about using or not using curly apostrophes and quotes?

Reply:
@pagustinqt20 @BrianMaki @sslaws For discussion.